.github/workflows/generate-gql-types.yml

```diff
@@ -0,0 +1,58 @@
+name: Generate and Commit GQL Types
+
+on:
+  schedule:
+    # Run the workflow every Monday at midnight (UTC)
+    - cron: '0 0 * * MON'
```

Comment:
@ermish It isn't triggered on a PR. That usually looks like this I though in `yml` based on our other workflows as follows

```
on:
  schedule:
    - cron: '0 0 * * MON'
  pull_request:
    branches:
      - main
```

This should only run as scheduled once a week and act as a bot similar to our renovate bot.

Reply:
Ohhh hmm I was thinking the cron job scheduler was triggered by a PR. Well it looks good then if it auto runs independently!